### DISCUSSION TITLE
Please tell me your favorite Youtuber

### DISCUSSION THREAD

**Discussion by pgDora56** [NAIST-SE/AttractingContributors#202] [General]
What's your favorite Youtuber? Please tell me!

Major Youtubers, Youtubers who play games, and Virtual Youtubers are all welcome.
Please feel free to add more.

https://github.com/pgDora56/AttractingContributors-Youtuber

**Comment by t-matsumoto4758** [comment]
Added!
please check and help me!
https://github.com/t-matsumoto4758/AttractingContributors-YourHometown

**Reply by pgDora56** [reply]
Thanks for your help!
I've sent a PR to your repository as well. Please check it.

**Comment by TakumiHiraide** [comment]
Added!
Please check and help me!!
https://github.com/TakumiHiraide/My-favorite-movie

**Reply by pgDora56** [reply]
Thanks for your help!
I would like to help, but your repository seems to be made in the wrong way. I recommend you to check it again.
